components/Select.tsx

```
@@ -100,22 +100,35 @@ const Select = forwardRef<HTMLDivElement, SelectProps>(
     const [showOptions, setShowOptions] = useState<boolean>(false);
     const [searchValue, setSearchValue] = useState("");
     const selectRef = useRef<HTMLDivElement>(null);
     const searchInputRef = useRef<HTMLInputElement | null>(null);
 
-    useOnClickOutside(selectRef as React.RefObject<HTMLElement>, () =>
-      setShowOptions(false)
-    );
+    useEffect(() => {
+      const handleWheel = (e: WheelEvent) => {
+        const optionsElement = selectRef.current?.querySelector(".options");
+        if (optionsElement && optionsElement.contains(e.target as Node)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)